tests/docs/pytest.org.md

```diff
@@ -71,29 +89,27 @@ def test_notopo():
 Test run
 
 ```
-py.test --inventory inv --host-pattern dut1 --module-path ../ansible/library/ --testbed tb --testbed_file tb.csv --topology t1 test_topo.py test_notopo.py -rA
-
-platform linux2 -- Python 2.7.12, pytest-4.6.9, py-1.8.1, pluggy-0.13.1
+py.test --inventory ../ansible/str --host-pattern str-7260cx3-acs-1 --module-path ../ansible/library/ --testbed vms7-t0-7260-1 --testbed_file ../ansible/testbed.csv --junit-xml=tr.xml --log-cli-level warn --topology t1 -k 'not test_restart_syncd' test_topo
```

Comment:
specify dummy names for inventory, hostname, tb, tb_file

Reply:
Done. Thanks!